_pages/en/strategy-technology-management.md

```diff
@@ -0,0 +1,338 @@
+
+<p>Investing in IT personnel means:</p>
+
+<ul>
+    <li>Reducing the reliance on commercial consulting firms</li>
+    <li>Perform a mandatory competency-based planning for all IT teams<sup><a href="#ref-13">13</a></sup></li>
+    <li>Implement a promotion program that can bridge IT personnel from IT-02 to IT-05</li>
+    <li>Create a rotational program that enables IT staff to move to and be exposed to other IT-related functions in ESDC</li>
+    <li>Streamlining HR hiring and <a href="https://gcmobility.gitlab.io/pages/about-gcmobility.html">mobility rules</a> to <a href="https://pipsc.ca/news-issues/outsourcing/part-one-real-cost-outsourcing/symptoms">incentive employment over contracting</a></li>
+    <li>Establishing <a href="https://sara-sabr.github.io/ITStrategy/strategy-transform-high-performing-org.html">clear improvement targets for all IITB teams</a> necessary to establishing stronger service management capabilities</li>
+    <li>Moving consultation budgets as learning budgets to fund the #1. Establish a Technology Literacy Program</li>
+    <li><a href="https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html">Empowering IT personnel</a> to deliver better services by providing them with the tools they need</li>
+</ul>
+
+<ol start="10" class="list-notes">
+<li id="ref-10" role="doc-endnote"><a href="https://www5.psc-cfp.gc.ca/dsad-dsda/time-to-staff/index-en.html">Source</a></li>
+<li id="ref-11" role="doc-endnote">Source: Data from ESDC and OCHRO</li>
+<li id="ref-12" role="doc-endnote"><a href="https://hranalytics-analytiquerh.tbs-sct.gc.ca/Classification/Home/ClassificationReports?GoCTemplateCulture=en-CA#ReportSectionTopDiv">Source</a></li>
+<li id="ref-13" role="doc-endnote">See <a href="https://014gc.sharepoint.com/:p:/r/sites/TechnicalDebt-Evergreening/Shared%20Documents/IT%20Evergreening/IT%20Evergreening%20Framework%20-%20Goals%20and%20Strategy/IT%20Evergreening%20-%20Strategy.pptx?d=w60b995bd1cd042628787b687d2205343&csf=1&web=1&e=mUziDY">IT Evergreening draft Strategy</a>, goals 8 and 9</li>
+</ol>
+    </td>
+  </tr>
+  <tr>
+    <td class="action-header"><strong>8. Apply Monitoring and Observability</strong></td>
+    <td rowspan="2">(removed)</td>
+  </tr>
+  <tr>
+    <td>
+<p><strong>Monitoring</strong> is tooling or a technical solution that allows teams to watch and understand the state of their systems. Monitoring is based on gathering predefined sets of metrics or logs.</p>
+
+<p><strong>Observability</strong> is tooling or a technical solution that allows teams to actively debug their system. Observability is based on exploring properties and patterns not defined in advance.</p>

```

Comment:
I've never heard this definition for observability! Do we have a source? Normally observability means watching without action.